src/distances.jl

```diff
@@ -104,15 +110,15 @@ function aniso2distance(semiaxes::AbstractVector, angles::AbstractVector;
   !rule.radian && (angles = deg2rad.(angles))
   _0 = zero(eltype(angles))
   N == 2 && (angles = [angles[1], _0, _0])
-  intr = @. (rule.motion == :CW) & !rule.extrinsic
+  intr = @. (rule.motion == :CW)  & !rule.extrinsic
```

Comment:
Extra space added by accident?

Reply:
Thanks. I'm without git in the machine I'm using this week. On monday I'll commit the adjusts